src/main/java/ay2122s1_cs2103t_w16_2/btbb/storage/JsonAdaptedOrder.java

```diff
@@ -38,6 +57,11 @@ public JsonAdaptedOrder(Order source) {
         clientName = source.getClientName().toString();
         clientPhone = source.getClientPhone().toString();
         clientAddress = source.getClientAddress().toString();
+        recipeName = source.getRecipeName().toString();
+        recipeIngredients = source.getRecipeIngredients().toJsonStorageString();
```

Comment:
This could have been replaced by using `JsonAdaptedIngredient.java` and storing it as a [json list](https://github.com/se-edu/addressbook-level3/blob/master/src/main/java/seedu/address/storage/JsonAdaptedPerson.java).
This way, whenever ingredient changes (e.g. we add more fields, we change user input format) we do not need to worry about changing RecipeIngredients toJsonStorageString method.

Then it would not rely on the fact that the `storage string format == user input string format`.
It would also remove the need for parsing in the model.